cmd/import.go

```diff
@@ -57,7 +58,10 @@ func importCmd(ctx *cli.Context) error {
 		}(secret.value)
 	}
 
-	creds, err := setCredentials(ctx, path, makers)
+	s, p := spinner("Attempting to set credentials...")
```

Comment:
In some places you're using `...` and others not, we should standardize the messages along with the ones coming back from the daemon.

Reply:
Agreed, missed this. Standard is no ellipses